pretend_file1.java

```diff
@@ -3,3 +3,11 @@
 java code original line 3 Edited DWS on desktop GBPC2185
 jave code original line 4 Edited MF on desktop GBPC2141
 jave code original line 5
+
+/**
+ * 
+ **/
+public interface pretend_fileq1 {
```

Comment:
are happy to set this an interface so we can implement this in different ways, for example a recursive or an iterative way